### ISSUE TITLE
reload is killing the diamond process

### ISSUE THREAD

**Issue by eserte** [python-diamond/Diamond#283]
Doing `/etc/init.d/diamond reload` would kill the diamond process. It seems that SIGHUP is not handled within diamond. So probably it's best to remove the "reload" action from the debian init script.

**Comment by themalkolm** [comment]
It does handle SIGHUP.

**Comment by eserte** [comment]
At least it was not handled in diamond 3.4.32:
```
$ sudo kill -HUP $(cat /var/run/diamond.pid)
$ sudo kill -HUP $(cat /var/run/diamond.pid)
kill: No such process
```
Was this fixed in the meanwhile? Maybe in commit 5e3a96bd?

**Comment by themalkolm** [comment]
Looks like.
